janitor/functions.py

```diff
@@ -133,6 +136,8 @@ def _remove_special(col):
 
     df = df.rename(columns=lambda x: re.sub("_+", "_", x))
     df = _strip_underscores(df, strip_underscores)
+    if original_columns:
```

Comment:
Suggest adding a comment line here, because it is logically connected with the previous change, but is not connected in contiguous lines.

```suggestion
    # Store the original column names, if it is enabled by user.
    if original_columns:
```